ngrinder-controller/src/main/java/org/ngrinder/security/NGrinderLdapContext.java

```diff
@@ -0,0 +1,65 @@
+package org.ngrinder.security;
+
+import lombok.RequiredArgsConstructor;
+import org.ngrinder.common.util.PropertiesWrapper;
+import org.ngrinder.infra.config.Config;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import javax.naming.Context;
+import javax.naming.NamingException;
+import javax.naming.ldap.InitialLdapContext;
+import java.util.Hashtable;
+
+import static org.ngrinder.common.constant.LdapConstants.*;
+
+@Component
+@RequiredArgsConstructor
+public class NGrinderLdapContext {
+	private static final Logger log = LoggerFactory.getLogger(NGrinderLdapContext.class);
```

Comment:
If you using lombok how about use `@Slf4j` instead of `LoggerFactory.getLogger(NGrinderLdapContext.class);`?

Reply:
Changed it and force pushed this branch